src/Foundatio.Redis/Queues/RedisQueue.cs

```diff
@@ -416,7 +415,7 @@ private async Task<RedisValue> DequeueIdAsync(CancellationToken linkedCancellati
                     timeout = timeout.TotalMilliseconds
                 }).AnyContext();
                 return result.ToString();
-            }, 3, TimeSpan.FromMilliseconds(100), _timeProvider, linkedCancellationToken, _logger).AnyContext();
+            }, linkedCancellationToken).AnyContext();
```

Comment:
@ejsmith I feel like this is the only callout needed in the entire pr. The retry interval for DequeueIdAsync is 100ms, this changes it to 1 second as that's the policy default.